### PR TITLE
VZ-7368: runbook for istio loadbalancer issue

### PR DESCRIPTION
**Pull request by neeraj-n-singh**
We have seen failure while creating loadbalancer for istio ingress gateway because no public subnets are available.
This runbook will take care of that.

### CODE REVIEW

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Istio Ingress Load Balancer Creation Issue  
+linkTitle: Istio Ingress Load Balancer Creation Issue
+description: Analysis detected Istio Ingress LoadBalancer is not created successfully.
+weight: 5
+draft: false
+---
+
+### Summary
+Analysis detected that the Verrazzano installation failed while creating loadbalancer for the Istio Ingress Gateway.
```
Comment (agv426) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md:10]: other advice docs have "load balancer" as 2 separate words

Comment (neeraj-n-singh) [reply content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md:10]: sure. I will fix all once other teammates share reviews as well.

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Istio Ingress Load Balancer Creation Issue  
+linkTitle: Istio Ingress Load Balancer Creation Issue
+description: Analysis detected Istio Ingress LoadBalancer is not created successfully.
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md:4]: RE-write: Analysis detected Istio ingress load balancer was not created successfully (no period required here)

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+weight: 5
+draft: false
+---
+
+### Summary
+Analysis detected that the Verrazzano installation failed while creating load balancer for the Istio Ingress Gateway.
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md:10]: creating load balancer -> creating the load balancer
Istio Ingress Gateway -> Istio ingress gateway (I noticed that Istio ingress gateway is not capitalized correctly in three places in this analysis doc, https://verrazzano.io/latest/docs/troubleshooting/diagnostictools/analysisadvice/istioingressgatewaynoloadbalancerip/. Would you mind fixing it there, too?)

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+The root cause will be while creating public load balancer there were no public subnet available.
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md:12]: Re-write: The root cause was that, while creating the public load balancer, there were no public subnets available.

**Review by rosemarymarano — COMMENTED** [review]
Few edits and suggestions

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md (hunk 1 above)
Comment (neeraj-n-singh) [reply content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md]: Done

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md (hunk 4 above)
Comment (rosemarymarano) [reply content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioLoadBalancerCreationIssue.md:12]: I don't see that this comment was implemented.

**Review by rosemarymarano — COMMENTED** [review]
One more. (Thanks for correcting Istio Ingress Gateway No External IP)

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/NginxLoadBalancerCreationIssue.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+title: Nginx Ingress Load Balancer Creation Issue
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/NginxLoadBalancerCreationIssue.md:2]: Nginx  -> NGINX (please change globally)

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/NginxLoadBalancerCreationIssue.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+linkTitle: Nginx Ingress Load Balancer Creation Issue
+description: Analysis detected Nginx ingress load balancer was not created successfully
+weight: 5
+draft: false
+---
+
+### Summary
+Analysis detected that the Verrazzano installation failed while creating the load balancer for the Nginx ingress controller.
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/NginxLoadBalancerCreationIssue.md:10]: Nginx ingress controller -> NGINX Ingress Controller

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/NginxLoadBalancerCreationIssue.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+---
+title: Nginx Ingress Load Balancer Creation Issue
+linkTitle: Nginx Ingress Load Balancer Creation Issue
+description: Analysis detected Nginx ingress load balancer was not created successfully
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/NginxLoadBalancerCreationIssue.md:4]: Nginx ingress load balancer -> NGINX Ingress Controller load balancer

**Review by rosemarymarano — COMMENTED** [review]
Few edits needed

**Review by rosemarymarano — COMMENTED** [review]
If you look at other VZ docs, you'll see the correct capitalization for NGINX Ingress Controller.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**